### PR TITLE
Feat/jfdi

### PR DESCRIPTION
**Pull request by cgmeyer**

### CODE REVIEW

File: gdcdictionary/examples/valid/pcr_assay_file.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "type":"pcr_assay_file",
+    "submitter_id":"pcr data file 1",
+	"analysis_software":"QuantaSoft",
+	"data_category":"Mutant Allele Fractions",
+	"data_type":"Tabulated Results",
+	"data_format":"CSV",
+	"experimental_strategy":"ddPCR",
+	"file_name":"results.csv",
+	"file_size":"12381660",
+	"md5sum":"d0880b77467c7782009520f8729cc50e",
```
Comment (dmiller15) [inline gdcdictionary/examples/valid/pcr_assay_file.json]: fix indentation

File: gdcdictionary/schemas/pcr_assay_file.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+uniqueKeys:
+  - [ id ]
+  - [ project_id, submitter_id ]
+
+properties:
+  $ref: "_definitions.yaml#/data_file_properties"
+
+  analysis_software:
+    description: >
+      The software used to process PCR assay raw data
```
Comment (dmiller15) [inline gdcdictionary/schemas/pcr_assay_file.yaml]: No period?

File: gdcdictionary/schemas/pcr_assay_file.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+id: "pcr_assay_file"
+title: PCR Assay File
+type: object
+namespace: https://www.bloodpac.org
+category: data_file
+program: '*'
+project: '*'
+description: >
+  A data file containing the results of a PCR assay
```
Comment (dmiller15) [inline gdcdictionary/schemas/pcr_assay_file.yaml:11]: No period?

File: gdcdictionary/schemas/pcr_assay_file.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    enum:
+      - "CSV"
+      - "QLB"
+      - "QLT"
+      - "QLP"
+      - "TSV"
+
+  experimental_strategy:
+    description: >
+      The PCR assay strategy used to generate the data file.
```
Comment (dmiller15) [inline gdcdictionary/schemas/pcr_assay_file.yaml]: This one has a period. Also there's an experimental strategy in the _terms file although its description says "sequencing strategy". Maybe generalize that and add the term here?

**Review by dmiller15 — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/pcr_assay_file.yaml (hunk 2 above)
Comment (cgmeyer) [reply gdcdictionary/schemas/pcr_assay_file.yaml]: *shrug* it's not a sentence ;)

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**